weave-scope/README.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: guides
+shorttitle: Using Weave Scope Standalone to Visualize and Monitor Docker Containers
+title: Using Weave Scope Standalone to Visualize and Monitor Docker Containers
+description: Use Weave Scope to monitor and visualize docker container clusters and swarms.
+tags: weave scope, docker container visibillity, cluster, swarms
+permalink: /guides/weave-scope/weave-scope-alone.html
+sidebarpath: /start/weave-scope-alone
+sidebarweight: 50
+---
+
+
+Weave Scope automatically detects and monitors every host, container and process in your infrastructure, builds a map showing their inter-communications and then presents an up-to-date view of your infrastructure in a web interface. You can visualize, monitor and control your distributed applications and troubleshoot bottlenecks, memory leaks or any other issues. It does this without requiring changes to your code or configuration, and without having to make declarations about your infrastructure that become out-of-date and stale. 
+
+While Weave Scope works with Docker and the Weave network, neither is required. Weave Scope can be deployed to any infrastructure, and works well in all cloud and bare-metal environments.
+
+There are two ways in which Weave Scope can be deployed: as a standalone configuration, or if you don't want to bother with the administration yourself, you can sign up for the Weave Scope cloud service.
+
+##What is Weave Scope Cloud Service?
```

Comment:
Scope consists of 3 components - the probe, the app and the UI.   In Cloud Mode, the probe runs on the users machine, the app on our infrastructure, and UI in the users browser. In hosted mode, the probe runs on the users machine, the app runs on the users machines, and the UI runs in the users browser.

This distinction is not clear to me from these 2 sections.